src/app_engine/constants.py

```diff
@@ -42,6 +42,21 @@ class EventType(object):
   # Event signifying that a room enters the state of having exactly
   # two participants.
   ROOM_SIZE_2 = 'room_size_2'
+  ICE_CONNECTION_STATE_CONNECTED = 'ice_connection_state_connected'
+
+
+class AnalyticsField(object):
```

Comment:
Yup ANALYTICS_FIELD_TYPE or something. Should be same number of characters. Keeping consistent with the rest of our constants.